### DISCUSSION TITLE
py-capellambse 0.5.5

### DISCUSSION THREAD

**Discussion by khammoutene** [dbinfrago/py-capellambse#212] [Q&A] [answered]
Hello,

I have upgraded to 0.5.5 mainly to get access to the:

- the realized classes (https://github.com/DSD-DBS/py-capellambse/discussions/179) 
- the states actions (https://github.com/DSD-DBS/py-capellambse/discussions/170)

When I "print" a class or a state, I don't see any new information.

By the way, I am executing the notebook via Capella (which launch a browser window with the notebook execution).

Is there a way to know, within the notebook, which py-capellambse I am running?

Thank you.

**Comment by Wuestengecko** [comment, marked as answer]
> Is there a way to know, within the notebook, which py-capellambse I am running?

Take a look at `capellambse.__version__`.

If it picked up the wrong version, you may also want to look at `capellambse.__file__` to see where it came from.

**Reply by khammoutene** [reply]
Thank you.
I did a full C: drive search for `capellambse.` but couldn't find files named `capellambse.__version__` nor `capellambse.__file__`

**Reply by Wuestengecko** [reply]
These aren't file names, these are Python attributes. Type that into the notebook :)
